src/address/types.rs

```diff
@@ -237,6 +237,8 @@ pub enum AddressError {
     InvalidUser,
     /// Invalid email domain
     InvalidDomain,
+    /// Fallback
+    Invalid(String),
```

Comment:
In order not to break everything I added a variant here for the `chumsky` parsing error. If you have a better idea let me know!